src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -30,3 +34,11 @@ export const AccountDetailsPage: FC = () => {
     </PageLayout>
   )
 }
+
+export const AccountDetailsView: FC<{
+  isLoading: boolean
+  account: RuntimeAccount | undefined
+  roseFiatValue: number | undefined
+}> = ({ isLoading, account, roseFiatValue }) => {
+  return <Account account={account} isLoading={isLoading} roseFiatValue={roseFiatValue} />
```

Comment:
What's a point of `AccountDetailsView` wrapper in general?

Reply:
I wanted a consistent pattern to reuse on search: AccountDetailsView, BlockDetailView, TransactionDetailView, since search should always almost match detail pages